--- libfpx-1.2.0.orig/basics/a_file.h	Sat Dec 23 23:10:00 2000
+++ libfpx-1.2.0/basics/a_file.h	Fri Mar 23 21:19:16 2001
@@ -46,12 +46,10 @@
 //  Global Functions
 //  ________________
 
-#ifdef _WINDOWS
 uint16          shortswap(uint16);  // Swap for Big/Little Endian
 uint32          longswap(uint32);   // Swap for Big/Little Endian
 #  define int32swap   longswap
 #  define int16swap   shortswap
-#endif
 
 //  Classes Declarations
 //  --------------------
@@ -310,7 +308,6 @@
 
 #ifdef DEFINE_INT32_FUNCTIONS
 
-#ifdef _WINDOWS
 inline uint32   longswap( uint32 x )
 {
   uint32 a, b;
@@ -318,7 +315,6 @@
   b = x & 0x0000ffff;
   return ( ((uint32)shortswap((uint16)b) << 16) | (uint32)shortswap((uint16)a) );
 }
-#endif
 
 inline Boolean Fichier::Ecriture(int32 ceLong)
 {
@@ -376,7 +372,6 @@
 
 
 // Added by GDN
-#ifdef _WINDOWS
 inline uint16   shortswap( uint16 x )
 {
   uint16 a, b;
@@ -384,7 +379,6 @@
   b = x & 0x00ff;
   return ( (b << 8) | a );
 }
-#endif
 
 inline Boolean Fichier::Ecriture(short ceShort)
 {
@@ -489,7 +483,8 @@
   Boolean ret;
 
 #ifndef __bigEndian
-  for (int32 i = 0; i < nshort; i++)
+  int32 i;
+  for (i = 0; i < nshort; i++)
     array[i] = shortswap (array[i]);
 #endif
 
@@ -510,7 +505,8 @@
 {
   Boolean ret;
 #ifndef __bigEndian
-  for (int32 i = 0; i < t; i++)
+  int32 i;
+  for (i = 0; i < t; i++)
     array[i] = int32swap (array[i]);
 #endif
 
